temporal-sdk/src/main/java/io/temporal/activity/LocalActivityOptions.java

```diff
@@ -81,18 +89,27 @@ public Builder setScheduleToCloseTimeout(Duration timeout) {
     }
 
     /**
-     * Maximum time to wait between retries locally, while keeping the Workflow Task open via a
-     * Heartbeat. If the delay between the attempts becomes larger that this threshold, a Workflow
-     * Timer will be scheduled. Default value is Workflow Task Timeout multiplied by 3.
+     * Time that the Activity Task can stay in the Task Queue before it is picked up by a Worker.
```

Comment:
```suggestion
     * Time that the Activity Task can stay in the worker's internal queue of Local Activities.
```

I think this could be misread to imply there's interaction with a real Task Queue here.